### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by chalhoub37** [frankvHoof93/TikTokLiveSharp#86]
HttpRequestException: Request was unsuccessful [403]
TikTokLiveSharp.Client.HTTP.TikTokHttpRequest+<GetContent>d__29.MoveNext () (at Library/PackageCache/dev.vanhoof.tiktokliveunity@18ae546fb8/TikTokLiveSharp/Client/HTTP/TikTokHttpRequest.cs:255)

i am trying the sample example , tried to add different live username but i am getting this error can someone help me how to fix it thank you

**Comment by chalhoub37** [comment]
> Currently in the TikTokLiveManager - Client Settings, if you tick the 'Skip Room Info' option, it will successfully connect to the livestream. However, skipping the Room Info may lead to a problem where it will respond with a 'Connected' event even if the stream ID is offline.

yes with checking 'skip room info' it can connect , can i know what is the disadvantage of skipping room info , what problem can i face ?

**Comment by NguyenVuNguyenPhys** [comment]
In this version of the library, it returns the connected status even if you or someone else is not livestreaming. So, if they are not livestreaming and the status is connected, then no event runs to your app. Even if that ID starts livestreaming later, your app still won't receive any comments and so on.
 If the host ID is online and livestreaming, then it works fine.

**Comment by 1fini** [comment]
> > Currently in the TikTokLiveManager - Client Settings, if you tick the 'Skip Room Info' option, it will successfully connect to the livestream. However, skipping the Room Info may lead to a problem where it will respond with a 'Connected' event even if the stream ID is offline.
> 
> yes with checking 'skip room info' it can connect , can i know what is the disadvantage of skipping room info , what problem can i face ?

Hi guys, what is this TikTokLiveManager you're talking about? Is it a software with GUI ? I'm having the same problem as everyone with the 403 error but I don't understand how you check the "Skip room info" option. 

Regards

**Comment by 1fini** [comment]
> In this version of the library, it returns the connected status even if you or someone else is not livestreaming. So, if they are not livestreaming and the status is connected, then no event runs to your app. Even if that ID starts livestreaming later, your app still won't receive any comments and so on. If the host ID is online and livestreaming, then it works fine.

Thanks for the information! It works
